### PR TITLE
Remove separate bootloader

### PR DESCRIPTION
**Pull request by gsilvis**
And debugging still works!

### CODE REVIEW

File: kernel/kernel.ld [hunk 1]
```diff
@@ -1,56 +1,47 @@
...
+
+    .rodata : AT(ADDR(.rodata) - KERNEL_BASE) {
+        *(.rodata)
     }
 
-    .eh_frame_hdr BLOCK(4K) : ALIGN(4K)
-    {
-        _eh_frame_hdr_start = .;
-        *(.eh_frame_hdr)
+    .eh_frame : AT(ADDR(.eh_frame) - KERNEL_BASE) {
```
Comment (allanlw) [inline kernel/kernel.ld]: do back traces still work?

File: kernel/kernel.ld [hunk 2]
```diff
@@ -1,56 +1,47 @@
...
 
-    .eh_frame BLOCK(4K) : ALIGN(4K)
-    {
-        _eh_frame_start = .;
-        *(.eh_frame)
-        _eh_frame_end = .;
+    . += KERNEL_BASE;
+
+    .text : AT(ADDR(.text) - KERNEL_BASE) {
+/*        *(.multiboot) */
```
Comment (allanlw) [inline kernel/kernel.ld]: delete this comment.

File: kernel/george.c [hunk 3]
```diff
@@ -58,23 +58,37 @@ typedef struct {
...
       (uint32_t *)phys_to_virt((uint64_t)mboot_struct_addr);
-  memtop = ((uint64_t)mboot_struct[2]) * 1024;
-  memtop += 0x100000;
+  uint64_t memtop = ((uint64_t)mboot_struct[2]) * 1024 + 0x100000;
   clear_screen();
   puts("Welcome to GeorgeOS, Multiboot Edition!\r\n");
-  initialize_allocator();
+
+  /* Find the end of the last multiboot module */
+  uint64_t lowest_safe_physical_address = (uint64_t)&_end - 0xFFFFFFFF80000000;
```
Comment (allanlw) [inline kernel/george.c]: It makes me sad how many places this constant shows up :(

File: kernel/kernel.ld (hunk 2 above)
Comment (gsilvis) [inline kernel/kernel.ld]: Done

File: kernel/kernel.ld (hunk 1 above)
Comment (gsilvis) [inline kernel/kernel.ld]: uh... no.

File: kernel/george.c (hunk 3 above)
Comment (gsilvis) [inline kernel/george.c]: What's more, there are two similar looking constants like it... it's definitely sad.

**Comment by allanlw** [comment]
Also, does this fix #3?

**Comment by gsilvis** [comment]
This fixes #3, I believe.  I do link 32-bit code inside a 64-bit elf, but that is significantly safer, I believe.

**Comment by allanlw** [comment]
This should not be merged until back tracing is fixed.

**Comment by gsilvis** [comment]
Backtracing appears to be a different bug.

**Pull request merged**